src/index.ts

```diff
@@ -9,6 +9,7 @@ import {
 } from '@unified-latex/unified-latex-util-arguments';
 import { typstEnvs, typstMacros, typstStrings } from './macros.js';
 import type { IState, LatexNode, StateData } from './types.js';
+import { runTransforms } from './transforms.js';
 
 export function parseLatex(value: string) {
   const file = unified()
```

Comment:
This should be a place to add overset/underset in latex. Instead of a trasnsform.